build/cluster-operator-ansible/cluster-api-prep/deploy-cluster-api.yaml

```diff
@@ -0,0 +1,152 @@
+---
+- name: Setup the master node group
+  hosts: localhost
+  tasks:
+  - import_role:
+      name: openshift_aws
+      tasks_from: setup_master_group.yml
+
+#- name: set the master facts for hostname to elb
+#  hosts: masters
+#  gather_facts: no
+#  remote_user: root
+#  tasks:
+#  - import_role:
+#      name: openshift_aws
+#      tasks_from: master_facts.yml
+
+#- name: run the init
+#  import_playbook: ../init/main.yml
+
+- name: install cluster-api
+  hosts: masters[1]
+  gather_facts: no
+  remote_user: root
+  vars:
+    cluster_api_namespace: kube-cluster
+  tasks:
+  - name: import lib_openshift
+    import_role:
+      name: lib_openshift
+
+  - name: slurp local template content
+    slurp:
+      src: files/cluster-api-template.yaml
+    delegate_to: localhost
+    register: template
+
+  - name: "create {{ cluster_api_namespace }} namesapce"
+    oc_project:
+      state: present
+      name: "{{ cluster_api_namespace }}"
+      display_name: "Cluster API"
+      node_selector:
+      - node-role.kubernetes.io/master=true
+
+  - name: "create cluster-api controller manager service account"
+    oc_serviceaccount:
+      namespace: "{{ cluster_api_namespace }}"
+      state: present
+      name: cluster-api-controller-manager
+      #register: service_account_results
+
+  - name: "create cluster-api service account"
+    oc_serviceaccount:
+      namespace: "{{ cluster_api_namespace }}"
+      state: present
+      name: cluster-api-apiserver
+    register: service_account_results
+
+  - name: save secret with token
```

Comment:
it looks like we don't need this token after all, so you can get rid of these tasks.
Because the pods run as the service accounts themselves, they get the token mounted automatically, we don't need to explicitly provide it.